package.json

```diff
@@ -29,7 +29,8 @@
     "test": "jest",
     "test:watch": "jest --watch",
     "test:coverage": "jest --coverage",
-    "dev": "start-storybook -p 6006"
+    "dev": "start-storybook -p 6006",
+    "version": "auto-changelog -p && git add CHANGELOG.md"
```

Comment:
This ties the auto-changelog generation to the 'npm version' command. 
See https://www.npmjs.com/package/auto-changelog#what-you-might-do-if-youre-clever

Reply:
Smart.